dot/network/transaction_test.go

```diff
@@ -29,6 +29,7 @@ func TestDecodeTransactionHandshake(t *testing.T) {
 	require.Equal(t, testHandshake, msg)
 }
 
+//go:generate mockgen -destination=mock_transaction_handler.go -package $GOPACKAGE . TransactionHandler
```

Comment:
Same question here, can you have it as a test file so it's not exported out of this package?